### DISCUSSION TITLE
fatal: repository 'https://github.com/vercel/next.js/tree/canary/examples/cms-cosmic/' not found

### DISCUSSION THREAD

**Discussion by Forwardslash-development** [vercel/next.js#19919] [Help]
# Bug report
Installing...
exec error: Error: Command failed: git clone https://github.com/vercel/next.js/tree/canary/examples/cms-cosmic nextjs-static-blog
Cloning into 'nextjs-static-blog'...
fatal: repository 'https://github.com/vercel/next.js/tree/canary/examples/cms-cosmic/' not found

## Describe the bug

// Install Cosmic CLI
npm i cosmic-cli -g

// Login
cosmic login

// Install app content to your Bucket and download codebase locally
cosmic install-app nextjs-static-blog

#ERROR
Installing...
exec error: Error: Command failed: git clone https://github.com/vercel/next.js/tree/canary/examples/cms-cosmic nextjs-static-blog
Cloning into 'nextjs-static-blog'...
fatal: repository 'https://github.com/vercel/next.js/tree/canary/examples/cms-cosmic/' not found

## System information

- OS: [WSL - Ubuntu20]

- Version of Node.js: [e.g. 14.9.0]

**Comment by tonyspiro** [comment]
Thank you @Forwardslash-development we are currently working on an update to the CLI example that fixes this. In the meantime, you can follow the steps to get started [from the repository](https://github.com/vercel/next.js/tree/canary/examples/cms-cosmic/#how-to-use).
